framework/src/mfem/bcs/MFEMScalarFunctorBoundaryIntegratedBC.C

```diff
@@ -0,0 +1,45 @@
+#ifdef MFEM_ENABLED
+
+#include "MFEMScalarFunctorBoundaryIntegratedBC.h"
+
+registerMooseObject("MooseApp", MFEMScalarFunctorBoundaryIntegratedBC);
+
+InputParameters
+MFEMScalarFunctorBoundaryIntegratedBC::validParams()
+{
+  InputParameters params = MFEMIntegratedBC::validParams();
+  params.addClassDescription("Adds the domain integrator to an MFEM problem for the linear form "
+                             "$(f, v)_\\Omega$ "
+                             "arising from the weak form of the forcing term $f$.");
+  params.addRequiredParam<MFEMScalarCoefficientName>(
+      "coefficient",
+      "The coefficient which will be used in the integrated BC. A coefficient can be any of the "
+      "following: a "
+      "variable, an MFEM material property, a function, or a post-processor.");
+  return params;
+}
+
+MFEMScalarFunctorBoundaryIntegratedBC::MFEMScalarFunctorBoundaryIntegratedBC(
+    const InputParameters & parameters)
+  : MFEMIntegratedBC(parameters),
+    _coef_name(getParam<MFEMScalarCoefficientName>("coefficient")),
+    _coef(getScalarCoefficient(_coef_name))
+{
+}
+
+// Create a new MFEM integrator to apply to the RHS of the weak form. Ownership managed by the
+// caller.
+mfem::LinearFormIntegrator *
+MFEMScalarFunctorBoundaryIntegratedBC::createLFIntegrator()
+{
+  return new mfem::BoundaryLFIntegrator(_coef);
```

Comment:
Is this memory ultimately managed in the MFEM library? If not, it would be nice to eventually change to use smart pointers

Reply:
Yes, MFEM will take ownership of this.